builds/cmake/Modules/FindHarfbuzz.cmake

```diff
@@ -55,7 +55,11 @@ if(HARFBUZZ_FOUND)
 
 	if(NOT TARGET Harfbuzz::Harfbuzz)
 		add_library(Harfbuzz::Harfbuzz UNKNOWN IMPORTED)
-		find_library(Freetype REQUIRED)
+
+		if(NOT TARGET Freetype::Freetype)
+			find_library(Freetype REQUIRED)
```

Comment:
could you change this to ``find_package`` pls?

Reply:
Sure. Done in 199c70c52fb1993a47f04382d43f753282cedace.